From 1869e5e7396795ed031d414ff1904c73b8a7e785 Mon Sep 17 00:00:00 2001
From: Wolf <wolf@wolfsden.cz>
Date: Thu, 16 Feb 2017 01:52:50 +0100
Subject: [PATCH 3/9] Fix config

---
 xbase64-3.1.2/xbase64-config.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xbase64-3.1.2/xbase64-config.in b/xbase64-3.1.2/xbase64-config.in
index 5aa75de..2db32d0 100755
--- a/xbase64-3.1.2/xbase64-config.in
+++ b/xbase64-3.1.2/xbase64-config.in
@@ -5,7 +5,7 @@ exec_prefix=@exec_prefix@
 exec_prefix_set=no
 CC="@CC@"
 CXX="@CXX@"
-LD="@SHARED_LD@"
+LD="@LD@"
 
 usage="\
 Usage: xbase-config [--prefix[=DIR]] [--exec-prefix[=DIR]] [--version] [--libs] [--cflags] [--cc] [--cxx] [--ld]"
-- 
2.11.1

